envs/env.py

```diff
@@ -450,4 +479,4 @@ def step(self, action) -> GymStepReturn:
         return self.state, reward, terminated, info
 
     def render(self, mode: str = "console") -> None:
-        pass
+        pass
```

Comment:
Missing newline at the end of file

https://thoughtbot.com/blog/no-newline-at-end-of-file